NotificationTrigger/time.test.js

```diff
@@ -0,0 +1,8 @@
+const { verifyNoticiableTime } = require('./time')
+describe('time test', () => {
+	it('test the noticiable time', () => {
```

Comment:
@shrijan-siris it's not a good practice to be returning different data types from same function. It would be hard to guess by just looking at the function name.

Reply:
can you elaborate it @syuraj